examples/microsoft-auth-provider/web-app/src/index.tsx

```diff
@@ -26,7 +26,9 @@ function LoginInfo(): JSX.Element {
     return (
       <>
         <div style={{ color: 'red' }}>An error occurred during authentication: {error}</div>
-        <button onClick={() => logOut()}>Logout</button>
+        <button type={'button'} onClick={() => logOut()}>
```

Comment:
Surely `type='button'` is simpler than `type={'button'}`?
```suggestion
        <button type='button' onClick={() => logOut()}>
```